app/assets/stylesheets/components/_artist-card.scss

```diff
@@ -0,0 +1,47 @@
+.container-artist-cards {
+  display: flex;
+  justify-content: space-around;
+  flex-direction: row;
+
+  .artist-card .artist-card-infos {
+    display: flex;
+    position: relative;
+  }
+
+  .artist-card-infos {
+    position: absolute;
+    flex-direction: column;
+    text-align: left;
+    position: relative;
+
+    .favorite-icon {
+      position: absolute;
+      box-shadow: 0 0 15px rgba(0,0,0,0.1);
+      right: 20px;
+      top: -65px;
+      pointer-events: none;
+    }
+
+    h3 {
+      margin-top: 8px;
+    }
+
+    p {
```

Comment:
does the css for p tag effects all the p tags in other pages ?

Reply:
Nope, only the p which are nested in the artist-card-infos :)
The one that affects all the p (unless it's nested somewhere in another class which has another rule) is in the component _paragraph.scss